src/components/screens/Overviews.js

```diff
@@ -322,19 +330,21 @@ export const Overviews = ({ navigation, route }) => {
       {query === QUERY_TYPES.POINTS_OF_INTEREST &&
         (switchBetweenListAndMap == SWITCH_BETWEEN_LIST_AND_MAP.TOP_FILTER ||
           showFilterByOpeningTimes) && (
-          <View>
+          <>
             {switchBetweenListAndMap == SWITCH_BETWEEN_LIST_AND_MAP.TOP_FILTER && (
               <IndexFilterWrapperAndList filter={filterType} setFilter={setFilterType} />
             )}
             {showFilterByOpeningTimes && (
-              <OptionToggle
-                label={texts.pointOfInterest.filterByOpeningTime}
-                onToggle={() => setFilterByOpeningTimes((value) => !value)}
-                value={filterByOpeningTimes}
-              />
+              <WrapperHorizontal>
+                <OptionToggle
+                  label={texts.pointOfInterest.filterByOpeningTime}
+                  onToggle={() => setFilterByOpeningTimes((value) => !value)}
+                  options={{ bold: true }}
+                  value={filterByOpeningTimes}
+                />
+              </WrapperHorizontal>
             )}
-            <Divider />
-          </View>
+          </>
         )}
       {query === QUERY_TYPES.POINTS_OF_INTEREST && showMap ? (
         <LocationOverview
```

Comment:
Normally I think the pin should be shown when switching to the map view.

https://github.com/user-attachments/assets/ef58c7f1-82e8-4c4a-8ecd-b35f32a50808

Reply:
they are shown in my case, but not centered and zoomed correctly to

<img width="379" alt="image" src="https://github.com/user-attachments/assets/cb2fc044-0370-48dd-9171-5f9d13b24257">

Reply:
same for the 3 "Bildung" pois from your clip

<img width="382" alt="image" src="https://github.com/user-attachments/assets/9615f1be-4b5c-46d7-a73e-24527bc68330">